src/Microsoft.AspNetCore.OData/Routing/ODataActionSelector.cs

```diff
@@ -10,14 +11,18 @@
 using Microsoft.AspNetCore.Mvc.Abstractions;
 using Microsoft.AspNetCore.Mvc.Controllers;
 using Microsoft.AspNetCore.Mvc.Infrastructure;
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+using Microsoft.AspNetCore.Mvc.ModelBinding.Binders;
+using Microsoft.AspNetCore.Mvc.Routing;
 #if NETSTANDARD2_0
     using Microsoft.AspNetCore.Mvc.Internal;
     using Microsoft.AspNetCore.Routing;
     using Microsoft.Extensions.Logging;
 #else
-    using Microsoft.AspNetCore.Routing;
+using Microsoft.AspNetCore.Routing;
```

Comment:
Seems to be imported either way. any reason we have this in the else?

Reply:
Good point.